error-prone-contrib/src/main/java/tech/picnic/errorprone/bugpatterns/Slf4jLoggerDeclaration.java

```diff
@@ -53,7 +53,7 @@ public final class Slf4jLoggerDeclaration extends BugChecker implements Variable
   private static final Matcher<ExpressionTree> IS_GET_LOGGER =
       staticMethod().onDescendantOf("org.slf4j.LoggerFactory").named("getLogger");
   private static final String CANONICAL_STATIC_LOGGER_NAME_FLAG =
-      "Slf4jLogDeclaration:CanonicalStaticLoggerName";
+      "Slf4jLoggerDeclaration:CanonicalStaticLoggerName";
```

Comment:
At some point we should likely write an error prone check that validates this :)